src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -146,4 +151,12 @@ export class PredictiveTextStudioWorkerImpl
     };
     return this.storage.updateProjectData(storedData);
   }
+
+  saveKMPPackage(KMP: ArrayBuffer): Promise<void> {
+    return this.storage.saveCompiledKMPAsArrayBuffer(KMP);
+  }
+
+  async getKMPPackage(): Promise<KMPPackageData> {
```

Comment:
I think we should keep`KMPPackageData` as an internal data structure, especially since the only place that consumes it just does `kmp.package` :joy:

```suggestion
  async getKMPPackage(): Promise<ArrayBuffer> {
```

Reply:
Sure. What I thought before was we can store different types of KMP file, so I create the interface. But using ArrayBuffer is absolutely much simpler.